source/msam/test/test_settings.py

```diff
@@ -20,29 +20,27 @@ class TestSettings(unittest.TestCase):
     This class extends TestCase with testing functions
     """
 
+    def setUp(self):
+        from chalicelib import settings
+        settings.DYNAMO_RESOURCE.reset_mock()
```

Comment:
need reset for DYNAMO_CLIENT?

Reply:
no since DYNAMO_RESOURCE is the only mock in the test cases used for assertions.